.github/workflows/ruby.yml

```diff
@@ -52,7 +52,8 @@ jobs:
           PGUSER: postgres
           PGPASSWORD: postgres
 
-      - uses: actions/checkout@v4
+      - name: Check out code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
I guess that version comments in this format will allow Dependabot to update them? Per https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash :

> Adding a comment referring to the tag enables automated dependency updates, such as Dependabot or Renovatebot, to update it whenever a new release is detected. [You can see an example of this in action in the Node.js Security WG repository](https://github.com/nodejs/security-wg/pull/1009).